packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -89,21 +89,22 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     event DepositRelayed(
         uint64 depositId,
         address indexed sender,
+        address slowRelayer,
```

Comment:
added to enable the full state re-creation from events (otherwise we would need to poll data from the client)